### PR TITLE
move update_park_fences to update_park_fences_around_tile

### PR DESCRIPTION
**Pull request by wolfreak99**

### CODE REVIEW

File: src/openrct2/world/tile_inspector.c [hunk 1]
```diff
@@ -389,7 +389,7 @@ sint32 tile_inspector_surface_show_park_fences(sint32 x, sint32 y, bool showFenc
 		if (!showFences)
 			surfaceelement->properties.surface.ownership &= ~0x0F;
 		else
-			update_park_fences(x << 5, y << 5);
+			update_park_fences_around_tile(x << 5, y << 5);
```
Comment (rwjuk) [inline src/openrct2/world/tile_inspector.c]: Did you intend to replace this usage of `update_park_fences()`? If so, could you explain why, as it's not a 1-to-1 replacement as with the others.

File: src/openrct2/world/map.c [hunk 2]
```diff
@@ -3875,7 +3875,7 @@ void map_extend_boundary_surface()
 		newMapElement->base_height = z;
 		newMapElement->clearance_height = z;
 
-		update_park_fences(x << 5, y << 5);
+		update_park_fences_around_tile(x << 5, y << 5);
```
Comment (rwjuk) [inline src/openrct2/world/map.c]: This change too.

File: src/openrct2/world/tile_inspector.c (hunk 1 above)
Comment (wolfreak99) [inline src/openrct2/world/tile_inspector.c]: Unless it's intentional to not update the tiles around it, i figured it was an original mistake. either that, or it's not even necessary to do 5 calls as the others previously did.

Comment (rwjuk) [inline src/openrct2/world/tile_inspector.c]: It could very well be. What happens if you don't do the additional 'update around tile' calls?

Comment (Gymnasiast) [inline src/openrct2/world/tile_inspector.c]: The tile inspector is one of those places where it should not update surrounding tiles, as it's a precision tool.

Comment (rwjuk) [inline src/openrct2/world/tile_inspector.c]: Ahh yes, I didn't see it was in the tile inspector (which is an OpenRCT2 addition, so can't have original bugs!).

Comment (wolfreak99) [inline src/openrct2/world/tile_inspector.c]: image
It looks like some calls are actually necessary. but I have removed the around-tile calls to the ones that weren't originally updating the tiles around and will push in a second

**Comment by janisozaur** [comment]
Hmm, I'm a bit late to the party, but this could have used new network version.

I initially thought it would be fine after the change @rwjuk pointed out for reverted, but there is ever so slightly chance of causing a desync.

Original code updated fences in different order to what this PR does and if for whatever reason it failed to update one of the tiles, it would have been different from the remote. I'm not 100% sure such a case exists, I haven't checked the code that much, but since we agreed on incrementing it whenever we _think_ it could cause desyncs…

**Comment by Broxzier** [comment]
With the changes as they are now, the game logic hasn't changed. The only difference is that some tiles may be updated before others now, but that shouldn't cause any desyncs.

**Comment by janisozaur** [comment]
I had a closer look at the code and indeed my assumption was ungrounded.

**Pull request merged**